index.bs

```diff
@@ -135,8 +135,8 @@ Algorithm steps that say
 mean the same thing as:
 
 <emu-alg>
-1. If _x_ is an abrupt completion, then return a promise rejected with _x_.[[value]].
-1. Else if _x_ is a Completion Record, then let _x_ be _x_.[[value]].
+1. If _x_ is an abrupt completion, then return a promise rejected with _x_.[[Value]].
```

Comment:
`[[value]]` vs `[[Value]]`, resolved by using `[[Value]]` and `[[Key]]` everywhere.